gcc/rust/rust-location.h

```diff
@@ -34,6 +34,8 @@ class Location
 
   location_t gcc_location () const { return gcc_loc_; }
 
+  void set_offset (unsigned int offset) { gcc_loc_ += offset; }
```

Comment:
This doesn't set the offset, but it does increment it.
```suggestion
  void increment_offset (unsigned int offset) { gcc_loc_ += offset; }
```
The function in itself is also not "needed" since there are already operator overloads for the `Location` type. So you can do `loc += 1` instead of `loc.set_offset(1)` for example.

I'm not a fan of operator overloading, but this is what we do in other parts of the compiler so I'm fine with it. I'd use these APIs instead :)